server/README.md

```diff
@@ -61,9 +61,13 @@ The server executable can be built with `nix build .#cardano-trasaction-lib-serv
     - `application/json`
 
 - Bytes to hash as hexadecimal string (`application/json;charset=utf-8`, `application/json`):
+- Hashing method from PlutusTx to hash with; one of `Sha2_256`, `Sha3_256`, `Blake2b_256`
```

Comment:
I think requiring the method to be capitalized might lead to user errors (if for some reason they don't want to use the client call in `QueryM.Crypto`)